Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 36
+        "Patch": 38
```

Comment:
@SatishThareja - @kaadhina has it right. if anyone from a rtm agent upgrades they should get the new task thats why the bump up higher than m105